app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -616,6 +621,23 @@ void loadFromIntent(final Intent intent) {
                     uri = Uri.parse(SettingsStore.getInstance(this).getHomepage());
                 }
             }
+
+            if (extras.containsKey("hide_webxr_interstitial")) {
+                mHideWebXRIntersitial = extras.getBoolean("hide_webxr_interstitial", false);
+                if (mHideWebXRIntersitial) {
+                    setWebXRIntersitialState(WEBXR_INTERSTITIAL_HIDDEN);
+                } else {
+                    // `forced` state is the default.
+                    setWebXRIntersitialState(WEBXR_INTERSTITIAL_FORCED);
+                }
```

Comment:
I don't think you want this else, if the default state changes, this will over write it.

Reply:
Sounds good to me.